File: Desarrollo_Movil/AndroidStudio/Ejemplo1Spinner/app/src/main/java/com/example/ejemplo1spinner/MainActivity.java
package com.example.ejemplo1spinner;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.AdapterView;
import android.widget.Spinner;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {
    Spinner s;

    @Override
    public void onCreate(Bundle b){
        super.onCreate(b);
        setContentView(R.layout.activity_main);
        s = (Spinner) findViewById(R.id.xsp);
        s.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener(){
            public void onItemSelected(AdapterView<?> av, View v, int i, long l){
                Toast.makeText(MainActivity.this, s.getSelectedItem().toString(),
                        Toast.LENGTH_LONG).show();
            }

            public void onNothingSelected(AdapterView<?> arg0){ }
        });
    }
}

File: Desarrollo_Movil/AndroidStudio/KotlindPDF4/app/src/main/java/com/example/kotlindpdf4/MainActivity.kt
package com.example.kotlindpdf4

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.Toast

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val btn = findViewById<Button>(R.id.btnPrincipal)

        btn.setOnClickListener {
            Toast.makeText(this,"SOY UN LISTENER DE UNA LINEA! LAMBDA!", Toast.LENGTH_LONG).show()
        }
    }
}
